### PR TITLE
Corrected Konga Page files format, HTML assignment

### PR DESCRIPTION
**Pull request by akoladepaul**
Corrected Konga Page files format, HTML assignment and a current state of CSS project and e-commerce project.

### Task: <task name here>

I <clear and concise explanation>
I <clear and concise explanation>

### CODE REVIEW

File: Paul_Akolade/HTML/Html Form Elements.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+QUESTION 1
```
Comment (kenny-wealth) [inline Paul_Akolade/HTML/Html Form Elements.txt:1]: Remove spaces in file name. Also state specifically what Task this is

File: Paul_Akolade/Design Konga Reg Page/css/login.css [hunk 2]
```diff
@@ -0,0 +1,280 @@
+/*
```
Comment (kenny-wealth) [inline Paul_Akolade/Design Konga Reg Page/css/login.css:1]: Please remove spaces in folder names

File: Paul_Akolade/CSS/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# dufuna_codecamp
```
Comment (kenny-wealth) [inline Paul_Akolade/CSS/README.md:1]: Please move all your folders into the submissions folder

File: Paul_Akolade/Design Konga Reg Page/login.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (kenny-wealth) [inline Paul_Akolade/Design Konga Reg Page/login.html:1]: Page looks good 👍

**Review by kenny-wealth — CHANGES_REQUESTED** [review]

File: Paul_Akolade/CSS/README.md (hunk 3 above)
Comment (akoladepaul) [reply Paul_Akolade/CSS/README.md:1]: Please i don't fully understand you. As regards moving all folders to submissions folder.
As at now all my folders are in paul_akolade folder which is then in the submissions folder.

**Comment by ndanni** [comment]
Kindly move your Tasks to the submissions folder.

**Comment by ndanni** [comment]
Kindly move your task to the submissions folder. Your PR will be approved once that is done.

**Pull request closed without merging**